### PR TITLE
Improve code format and documentation

### PR DESCRIPTION
**Pull request by rhumbertgz**

### CODE REVIEW

File: mpy_robot_tools/ctrl_plus.py [hunk 1]
```diff
@@ -30,6 +30,9 @@
 
 
 class SmartHub:
+    """Class for ...
+
```
Comment (antonvh) [inline mpy_robot_tools/ctrl_plus.py:37]: SmartHub is a class for LEGO Power Functions 2.0 battery boxes like TechnicHub, CityHub and BoostHub. The class implements communication to and from these boxes using the lpf2 protocol over Bluetooth Low Energy (BLE).   LEGO has documented it here: https://lego.github.io/lego-ble-wireless-protocol-docs/

**Review by antonvh — COMMENTED** [review]
Added some extra info.

**Comment by antonvh** [comment]
Thanks for your very clear commit messages!

**Comment by antonvh** [comment]
Would you like to join our Slack channel? Email me on anton@antonsmindstorms.com.

**Comment by antonvh** [comment]
I've been rather reluctant to add long docstrings because they seem to increase the size of compile micropython libraries and I want to be careful with memory.

**Comment by antonvh** [comment]
But maybe I worried for nothing: https://docs.micropython.org/en/latest/develop/compiler.html?highlight=main

File: mpy_robot_tools/ctrl_plus.py (hunk 1 above)
Comment (rhumbertgz) [reply mpy_robot_tools/ctrl_plus.py:37]: Done

**Pull request closed without merging**